### ISSUE TITLE
Can this be made to use with `term2` instead?

### ISSUE THREAD

**Issue by serapath** [AustP/git-go#5]
There is a package [`term2`](https://atom.io/packages/term2) which allows to open a shell as an atom tab. So when I hit `ctrl-g ctrl-g` to open `git-go`, i would like to see a seperate tab (maybe even in another `pane`), which is just a normal `bash` on that file's path (with all the configuration i have customized), but augments it with some "helper tabs" that give descriptions about how to use git :-)

I'm observing a lot of "custom terminals" being made for `atom`, but after trying different approaches, i like `term2` the most, because in the future i might even be able to save them with my current projects layout

**Comment by AustP** [comment]
Hmm I hadn't seen term2 before. I just installed it and tested it out and ran into quite a few issues within the first few minutes. For example, I'm using a light color theme, and I had to tweak quite a few settings to make it fit with my theme / UI. Then I ran a `git status` command and the output broke the layout of atom (it didn't scroll properly). I had to restart atom to fix it.

However, I do agree with you that having a custom terminal inside of this package is not plausible for long term support. I would love to rip out the custom terminal and replace it with the "de facto" terminal package. The problem though of course is that there is no "de facto" terminal package yet. Perhaps term2 will get there, but in it's current state I'm gonna have to hold off on incorporating it in.

I'll leave this issue open and check back periodically to see if I can replace the custom terminal with something more standardized.

**Comment by serapath** [comment]
agree. i have the same feeling. a defacto terminal would be awesome.
- I imagine, i fire up my cross-platform editor (atom) and have a tree-view which indicated folders for which a custom layout (open files/tabs/panes) has been saved.
- Right-clicking (or hotkey) any folder would load a new atom-editor with that folder as root and the saved tabs/files layout.
- in the far future, maybe every "project" could even have a different set of plugins/themes that load with the new atom instance.

**The cool thing would be, that the `project settings` for each folder as a root, would not only save open files/tabs/panes (maybe themes/plugins), but also `terminal tabs`.**

While many people do this using a terminal with screen/tmux and some terminal based editors (vim/emacs/...) - this would all be html/js/css based with atom instead and gives you an awesome active ecosystem of plugins on github. Many nice things will come from that i believe :-)
